public/locales/fa/translation.json

```diff
@@ -160,8 +181,20 @@
     "install": {
         "path": "انتخاب محل نصب"
     },
+    "label": {
+        "playing": {
+            "start": "",
+            "stop": ""
+        },
+        "saves": {
+            "syncing": ""
+        }
+    },
+    "launch": {
+        "options": "Launch Options..."
+    },
     "library": {
-        "gridView": "نمای شبکه‌ای",
+        "gridView": "نمای شبکه\u200cای",
```

Comment:
Could you remove this `\u200c` manually? this is i18n bug with languages like this

Reply:
👍 https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/pull/1311/commits/3842ba3b58a7fcb42003ad4f47ffd56e356c8826